### PR TITLE
LibVA Updates - Support Public Version

### PR DESCRIPTION
**Pull request by kiritigowda**
* Find and link to public version of libva
* Support VA Version greater than 1.5.X
* Support link to AMDGPU VA headers if public unavailable
* Package link updates

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -190,16 +190,16 @@ if(HIP_FOUND AND Libva_FOUND)
   file(READ "/etc/os-release" OS_RELEASE)
   string(REGEX MATCH "22.04" UBUNTU_22_FOUND ${OS_RELEASE})
 
-  # Set the dependent packages
-  set(rocDecode_DEBIAN_PACKAGE_LIST  "rocm-hip-runtime, libva-amdgpu-drm2, libdrm-amdgpu1, mesa-amdgpu-va-drivers")
-  set(rocDecode_RPM_PACKAGE_LIST     "rocm-hip-runtime, libva-amdgpu, libdrm-amdgpu, mesa-amdgpu-dri-drivers")
+  # Set the dependent packages - TBD: mesa-amdgpu-va-drivers should bring libdrm-amdgpu, but unavailable on RPM
+  set(rocDecode_DEBIAN_PACKAGE_LIST  "rocm-hip-runtime, libva2, libdrm-amdgpu1, mesa-amdgpu-va-drivers")
+  set(rocDecode_RPM_PACKAGE_LIST     "rocm-hip-runtime, libva, libdrm-amdgpu, mesa-amdgpu-dri-drivers")
```
Comment (kiritigowda) [inline CMakeLists.txt:195]: @AryanSalmanpour -- libdrm-amdgpu should be a dependency of va-driver, this is stop-gap fix till va-drivers are available on RPM

**Comment by kiritigowda** [comment]
@eidenyoshida -- this PR will support all ROCm supported OS -- package deps for CI docker here --  https://github.com/ROCm/rocDecode/blob/f73799fc2b8ec94a6aeb32f5c2b5f6d7588d4797/rocDecode-setup.py#L124

**Comment by kiritigowda** [comment]
@AryanSalmanpour  -- our core deps vs ldd

```
+ objdump -x /opt/rocm/lib/librocdecode.so
+ grep NEEDED
  NEEDED               libva.so.2
  NEEDED               libva-drm.so.2
  NEEDED               libamdhip64.so.6
  NEEDED               libstdc++.so.6
  NEEDED               libm.so.6
  NEEDED               libgcc_s.so.1
  NEEDED               libc.so.6
+ ldd -v /opt/rocm/lib/librocdecode.so
	linux-vdso.so.1 (0x00007ffdf5fa9000)
	libva.so.2 => /opt/amdgpu/lib64/libva.so.2 (0x00007f59e1285000)
	libva-drm.so.2 => /opt/amdgpu/lib64/libva-drm.so.2 (0x00007f59e1280000)
	libamdhip64.so.6 => /opt/rocm-6.2.0-13597/lib/libamdhip64.so.6 (0x00007f59dfaf7000)
	libstdc++.so.6 => /lib64/libstdc++.so.6 (0x00007f59df8d0000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f59df7f5000)
	libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f59df7da000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f59df5cf000)
	libdrm.so.2 => /opt/amdgpu/lib64/libdrm.so.2 (0x00007f59df5b8000)
	libamd_comgr.so.2 => /opt/rocm-6.2.0-13597/lib/libamd_comgr.so.2 (0x00007f59d58a5000)
	libhsa-runtime64.so.1 => /opt/rocm-6.2.0-13597/lib/libhsa-runtime64.so.1 (0x00007f59d55ae000)
	libnuma.so.1 => /lib64/libnuma.so.1 (0x00007f59d55a0000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f59e132a000)
	libz.so.1 => /lib64/libz.so.1 (0x00007f59d5584000)
	libzstd.so.1 => /lib64/libzstd.so.1 (0x00007f59d54ad000)
	libtinfo.so.6 => /lib64/libtinfo.so.6 (0x00007f59d547e000)
	libelf.so.1 => /lib64/libelf.so.1 (0x00007f59d5463000)
	librocprofiler-register.so.0 => /opt/rocm-6.2.0-13597/lib/librocprofiler-register.so.0 (0x00007f59d53bd000)
	libdrm_amdgpu.so.1 => /opt/amdgpu/lib64/libdrm_amdgpu.so.1 (0x00007f59d53b0000)

```

File: CMakeLists.txt [hunk 2]
```diff
@@ -190,16 +190,16 @@ if(HIP_FOUND AND Libva_FOUND)
   file(READ "/etc/os-release" OS_RELEASE)
   string(REGEX MATCH "22.04" UBUNTU_22_FOUND ${OS_RELEASE})
 
-  # Set the dependent packages
-  set(rocDecode_DEBIAN_PACKAGE_LIST  "rocm-hip-runtime, libva-amdgpu-drm2, libdrm-amdgpu1, mesa-amdgpu-va-drivers")
-  set(rocDecode_RPM_PACKAGE_LIST     "rocm-hip-runtime, libva-amdgpu, libdrm-amdgpu, mesa-amdgpu-dri-drivers")
+  # Set the dependent packages - TBD: mesa-amdgpu-va-drivers should bring libdrm-amdgpu, but unavailable on RPM
```
Comment (kiritigowda) [inline CMakeLists.txt:193]: @arvindcheru -- we need to resolve mesa-amdgpu-va-drivers package

File: cmake/FindLibva.cmake [hunk 3]
```diff
@@ -21,14 +21,33 @@
...
+find_library(LIBVA_LIBRARY NAMES va HINTS /usr/lib/x86_64-linux-gnu /usr/lib64 /opt/amdgpu/lib/x86_64-linux-gnu /opt/amdgpu/lib64)
+find_library(LIBVA_DRM_LIBRARY NAMES va-drm HINTS /usr/lib/x86_64-linux-gnu /usr/lib64 /opt/amdgpu/lib/x86_64-linux-gnu /opt/amdgpu/lib64)
+find_path(LIBVA_INCLUDE_DIR NAMES va/va.h PATHS /usr/include /opt/amdgpu/include)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Libva DEFAULT_MSG LIBVA_INCLUDE_DIR LIBVA_LIBRARY)
 mark_as_advanced(LIBVA_INCLUDE_DIR LIBVA_LIBRARY LIBVA_DRM_LIBRARY)
 
+if(Libva_FOUND)
+  # Find VA Version
```
Comment (kiritigowda) [inline cmake/FindLibva.cmake:33]: @arvindcheru -- version we support

**Comment by kiritigowda** [comment]
@AryanSalmanpour @eidenyoshida  -- this will support `SLES SP4`+

**Review by AryanSalmanpour — APPROVED** [review]

**Pull request merged**